zerver/openapi/zulip.yaml

```diff
@@ -7955,6 +7979,203 @@ paths:
                         "msg": "",
                         "result": "success",
                       }
+  /settings/display:
+    patch:
+      operationId: update_display_settings
+      tags: ["users"]
+      description: |
+        This endpoint is used to edit user's display settings.
+
+        `PATCH {{ api_url }}/v1/settings/display`
+      parameters:
+        - name: twenty_four_hour_time
+          in: query
+          description: |
+            Set of display of time according to 24-hour clock notation.
+          schema:
+            type: boolean
+          example: true
+        - name: dense_mode
+          in: query
+          description: |
+            Set display in dense mode.
+            It is an experimental setting only available in development environments.
+          schema:
+            type: boolean
+          example: true
```

Comment:
I could not find this setting as well as `high_contrast_mode` setting available on czo, but these were available on development server. After going through various documentation and code related to these settings I found that `dense_mode` is an experimental settings only available in development environment. So I have added the same description for `dense_mode`. But as I could not find any such description for `high_contrast_mode` anywhere I have added similar description for this also. So any suggestions for change over documentation these two settings are highly awaited.